cmd/influx/cli/cli_test.go

```diff
@@ -473,6 +494,48 @@ func TestParseCommand_HistoryWithBlankCommand(t *testing.T) {
 func emptyTestServer() *httptest.Server {
 	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Influxdb-Version", SERVER_VERSION)
-		return
+
+		switch r.URL.Path {
+		case "/query":
+			values := r.URL.Query()
+			parser := influxql.NewParser(bytes.NewBufferString(values.Get("q")))
+			q, err := parser.ParseQuery()
+			if err != nil {
+				w.WriteHeader(500)
+				return
+			}
+			stmt := q.Statements[0]
+
+			switch stmt.(type) {
+			case *influxql.ShowDatabasesStatement:
+				out, _ := json.Marshal(map[string]interface{}{
+					"results": []map[string]interface{}{
+						map[string]interface{}{
+							"series": []map[string]interface{}{
+								map[string]interface{}{
+									"name":    "databases",
+									"columns": []string{"name"},
+									"values": []interface{}{
+										[]string{"db"},
+									},
+								},
+							},
+						},
+					},
+				})
+				w.WriteHeader(200)
+				w.Write(out)
```

Comment:
I figured it was easier to read since it was a Go structure instead of embedded JSON.

Since it's a test and the amount of difference is likely milliseconds, I didn't view it as a very important difference. I can change it though.